modules/exploits/windows/smb/ms10_061_spoolss.rb

```diff
@@ -37,6 +37,7 @@ def initialize(info = {})
       'License'        => MSF_LICENSE,
       'Platform'       => 'win',
       'Notes' => {
+        'AKA' => ['EMERALTHREAD'],
```

Comment:
Same as above, didn't see a reference to `EMERALTHREAD` in either link on the description.